web/profiles/custom/os2loop/modules/os2loop_settings/README.md

```diff
@@ -0,0 +1,78 @@
+# OS2Loop Settings
```

Comment:
Maybe add a section about why we need the additional configurable permission layer.